app/views/devise/passwords/edit.html.haml

```diff
@@ -0,0 +1,20 @@
+%main.container.general-main-wrap
+  .text-center.mb-25
+    %h2
+      = t('.password_edit')
+  .general-form.field
+    = simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f|
```

Comment:
Please read documentation https://github.com/plataformatec/simple_form and use this gem properly